_posts/2020-04-17-snowplow-ios-tracker-1.3.0-released/index.md

```diff
@@ -0,0 +1,309 @@
+    return YES;
+}
+
+- (NSArray<SPSelfDescribingJson *> *)generatorFromEvent:(id<SPInspectableEvent>)event {
+    return @[
+        [[SPSelfDescribingJson alloc] initWithSchema:@"iglu:com.snowplowanalytics.snowplow/test_sdj/jsonschema/1-0-1" andData:@{@"key": @"value"}],
+    ];
+}
+
+@end
+{% endhighlight %}
+
+It can be passed to the tracker as usual:
+
+{% highlight objc %}
+SPGlobalContext *contextGeneratorGC = [[SPGlobalContext alloc] initWithContextGenerator:[GlobalContextGenerator new]];
+[tracker addGlobalContext:rulesetGC tag:@"tag1"];
+{% endhighlight %}
+
+<h2 id="gdpr">2. Tracking GDPR basis for processing with the GDPR context</h2>
+
+This release introduces the `gdprContext` and the `enableGdprContext` methods, which append a GDPR context to all events once enabled ([#425][425]).
+This allows users to easily record the basis for data collection and relevant documentation, and enables a straightforward audit flow for all events.
+
+It takes the following arguments:
+
+|      **Name** | **Description**             | **Required?** | **Type** |
+|--------------:|:----------------------------|:--------------|:---------|
+|       `basis` | GDPR Basis for processing | Yes           | Enum String   |
+|     `documentId` | ID of a GDPR basis document     | No           | String   |
+|        `documentVersion` | Version of the document        | No            | String   |
+| `documentDescription` | Description of the document | No            | String   |
+
+The required basisForProcessing accepts only the following literals: `consent`, `contract`, `legal_obligation`, `vital_interests`, `public_task`, `legitimate_interests` - in accordance with the [five legal bases for processing](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/lawful-basis-for-processing/)
+
+The GDPR context is enabled by calling the `setGdprContextWithBasis:documentId:documentVersion:documentDescription:` method of the tracker builder or by calling the `enableGdprContextWithBasis:documentId:documentVersion:documentDescription:` method once the tracker has been initialised.
+
+It is called as follows:
+
+{% highlight objc %}
+SPTracker *tracker = [SPTracker build:^(id<SPTrackerBuilder> builder) {
+    [builder setEmitter:emitter];
+    ...
+    [builder setGdprContextWithBasis:SPGdprProcessingBasisConsent
+                          documentId:@"someId"
+                     documentVersion:@"0.1.0"
+                 documentDescription:@"a demo document description"];
+}];
+{% endhighlight %}
+
+<h2 id="spm">3. Swift Package Manager support</h2>
+
+The Swift Package Manager is the official Apple's dependency manager designed for Swift libraries. Since Xcode 11 SPM is compatible with the iOS build system contributing to enlarge the mobile developers interst around this new dependency manager that will compete against the largely longtime used Cocoapods and Carthage.
```

Comment:
```suggestion
The Swift Package Manager is the official Apple dependency manager designed for Swift libraries. Since Xcode 11, SPM is compatible with the iOS build system.
```